File: Antiguo/Lab09/TransitSoft/TransitSoftPersistance/dao/VehiculoDAO.cs
using System;
using System.ComponentModel;
using TransitSoft.Model;

namespace TransitSoft.Persistance.dao
{
    public interface VehiculoDAO
    {
        BindingList<VehiculoDTO> ListarPorPropietario(String descripcion_propietario);
    }
}

File: EX2_Flavio/TransitSoftPersistance/src/main/java/pe/pucp/transitsoft/dao/VehiculoDAO.java
package pe.pucp.transitsoft.dao;

import java.util.ArrayList;
import pe.pucp.transitsoft.model.VehiculoDTO;

public interface VehiculoDAO {
    
    public ArrayList<VehiculoDTO> listarPorPropietario(String descripion_propietario);
    public int insertar(VehiculoDTO vehiculo);
    public int modificar(VehiculoDTO vehiculo);
    public int eliminar(VehiculoDTO vehiculo);
    public VehiculoDTO obtenerPorId(Integer id);
    public ArrayList<VehiculoDTO> listarTodos();
}
